src/config/level/phpunit/phpunit-specific-method.yml

```diff
@@ -0,0 +1,6 @@
+rectors:
+    Rector\Rector\Contrib\PHPUnit\SpecificMethod\AssertTrueFalseToSpecificMethodRector: ~
+    Rector\Rector\Contrib\PHPUnit\SpecificMethod\AssertSameBoolNullToSpecificMethodRector: ~
+    Rector\Rector\Contrib\PHPUnit\SpecificMethod\AssertTrueFalseInternalTypeToSpecificMethodRector: ~
+    Rector\Rector\Contrib\PHPUnit\SpecificMethod\AssertSameCountToSpecificMethodRector: ~
+    Rector\Rector\Contrib\PHPUnit\SpecificMethod\AssertTrueIssetToObjectHasAttributeRector: ~
```

Comment:
I've placed this to own config, since it not PHPUnit 6.0 related, but rather "method naming" related